src/samples/ChildBenefitsClaim/cookiePage/index.tsx

```diff
@@ -5,18 +5,28 @@ import AppFooter from '../../../components/AppComponents/AppFooter';
 import CookiePageTable from './CookiePageTable';
 import { scrollToTop } from '../../../components/helpers/utils';
 import MainWrapper from '../../../components/BaseComponents/MainWrapper';
+import { useLocation } from 'react-router-dom';
+import { registerServiceName } from '../../../components/helpers/setPageTitleHelpers';
 
 const FIND_OUT_MORE_URL = 'https://www.tax.service.gov.uk/help/cookie-details';
 
 export default function CookiePage() {
   const { t } = useTranslation();
+  const location = useLocation();
   scrollToTop();
 
+  const searchParams = new URLSearchParams(location.search);
+  const appNameHeader = searchParams.get('appname');
+  const serviceNamePageTitle = searchParams.get('serviceNamePageTitle');
+  const serviceParam = searchParams.get('serviceParam');
+  
+  registerServiceName(t(serviceNamePageTitle));
+
   return (
     <>
-      <AppHeader appname={t('CLAIM_CHILD_BENEFIT')} hasLanguageToggle isPegaApp={false} />
+      <AppHeader appname={t(appNameHeader)} hasLanguageToggle isPegaApp={false} />
```

Comment:
So now on the cookies page I can change `http://localhost:3502/cookies?appname=CLAIM_CHILD_BENEFIT&serviceParam=463&serviceNamePageTitle=CLAIM_CHILD_BENEFIT` 

to this:
`http://localhost:3502/cookies?appname=**MYNEWTITLE**&serviceParam=463&serviceNamePageTitle=CLAIM_CHILD_BENEFIT`

This doesn't seem like it is the best approach

Reply:
![image](https://github.com/user-attachments/assets/cb136c68-891c-481d-b210-44391d17bdff)